src/Aspire.Dashboard/Resources/ControlsStrings.resx

```diff
@@ -128,7 +128,7 @@
   </data>
   <data name="EnvironmentVariablesFilterToggleShowSpecOnly" xml:space="preserve">
     <value>Show spec only</value>
-    <note>Spec means from the resource specification</note>
+    <comment>Spec means from the resource specification</comment>
```

Comment:
I fixed these early last week to be `<comment>` and then [they changed back to note](https://github.com/dotnet/aspire/pull/1332/files#diff-2453339c9205116de2ce41f77436813b3bbd248dd69d23ec85feaae87e55e23fR131).

What is changing them?

Reply:
@JamesNK I'm guessing this was due a botched merge commit by me, considering it just happened to this one file